engine/gamesys/src/gamesys/components/comp_collision_object.cpp

```diff
@@ -931,14 +927,18 @@ namespace dmGameSystem
                     // Give that the assumption above holds, this assert will hold too.
                     assert(world->m_ComponentTypeIndex == context->m_World->m_ComponentTypeIndex);
 
+                    dmMessage::URL* receiver = (dmMessage::URL*)malloc(sizeof(dmMessage::URL));
+                    dmMessage::ResetURL(receiver);
+                    receiver->m_Socket = dmGameObject::GetMessageSocket(collection);
+                    receiver->m_Path = dmGameObject::GetIdentifier(sender_instance);
+                    
                     dmPhysics::RayCastRequest request;
                     request.m_From = ddf->m_From;
                     request.m_To = ddf->m_To;
                     request.m_IgnoredUserData = sender_instance;
                     request.m_Mask = ddf->m_Mask;
                     request.m_UserId = component_index << 16 | (ddf->m_RequestId & 0xff);
-                    request.m_UserData = (void*)sender_instance;
-
+                    request.m_UserData = (void*)receiver;
```

Comment:
We can send URL instead of the instance. If object by URL doesn't exist, it will be just an error message, instead of crash